pennylane/plugins/default_qubit.py

```diff
@@ -356,24 +356,68 @@ def expval(self, observable, wires, par):
           float: expectation value :math:`\expect{A} = \bra{\psi}A\ket{\psi}`
             """
```

Comment:
These are inherited methods, so the docstring should not be filled out - instead, Sphinx will automatically use the base class docstring in `Device`. For maintainability, this makes it easier on our end as well - only one docstring will need to be updated for typos/change in functionality.